pipedrive/client-nodejs
# GRAL-1917: remove external fs package and package-lock.json

### Changes
* Removed external fs package
* Disabled creation of package-json file as it's not necessary for libraries (like this one) that are used within other projects

### Testing
* Install this branch: `npm install git://github.com/pipedrive/client-nodejs.git#GRAL-1917`
* Run the following code (you need your own file, of course :)) to verify upload still works (it uses the 'fs' package) and that things still run in general

```
const pipedrive = require('pipedrive');

pipedrive.Configuration.apiToken = 'xxx'; // your API key

(async () => {
	try {
		const dealId = 111; // some deal ID in your pipeline

		const result = await pipedrive.FilesController.addFile({
			file: 'file1.jpg',
			dealId,
		});

		console.log(result);
	} catch (error) {
		console.log(error);
	}
})();
```

### Related PRs
Based on this: https://github.com/pipedrive/client-nodejs/pull/184
